tag-expressions/go/parser_test.go

```diff
@@ -0,0 +1,106 @@
+package tagexpressions_test
+
+import (
+	"testing"
+
+	tagexpressions "."
```

Comment:
This is what was done in gherkin/go. This is never something I have done and think is too much against the go way of doing things. I would prefer we setup go to have its own docker image

Reply:
I think it's more pragmatic to add a dot in a test than maintaining several Docker images.

Reply:
It should also be possible to build the code without having to use Docker.